lom/src/pluginmgr/pluginmgr_common/pluginmgr_helper.go

```diff
@@ -778,46 +824,48 @@ func CreatePluginInstance(pluginID plugins_common.PluginId, actionCfg *lomcommon
 /*
  * setup UNIX signals
  */
-func SetupSignals() error {
+func SetupSignals() {
     signalChan := make(chan os.Signal, 1)
     signal.Notify(signalChan, syscall.SIGTERM)
 
     lomcommon.GetGoroutineTracker().Start("HandleSyslogSignal"+lomcommon.GetUUID(),
-        func() error {
+        func() {
```

Comment:
How will these goroutines exit. Guess it will block waitall for ever.

Why do we need goroutinetracker for these that run for the life of process

Reply:
one reason is during shutdown, it can print goroutines info. 
system shutdoen channel can be tied inside here to exit from goroutine.

Reply:
remove from tracker and make it go

Reply:
TODO